Yank/restraints.py

```diff
@@ -2576,12 +2656,13 @@ def restrain_state(self, thermodynamic_state):
         rmsd_cv = openmm.RMSDForce(self.reference_sampler_state.positions, restrained_atoms)
 
         # Create an CustomCVForce
-        energy_expression = 'lambda_restraints * step(dRMSD) * (K_RMSD/2)*dRMSD^2; dRMSD = (RMSD-RMSD0);'
+        energy_expression = ('{} * step(dRMSD) * (K_RMSD/2)*dRMSD^2; '
+                             'dRMSD = (RMSD-RMSD0);'.format(self.full_restraint_name))
         energy_expression += 'K_RMSD = %f;' % self.K_RMSD.value_in_unit_system(unit.md_unit_system)
         energy_expression += 'RMSD0 = %f;' % self.RMSD0.value_in_unit_system(unit.md_unit_system)
         restraint_force = openmm.CustomCVForce(energy_expression)
         restraint_force.addCollectiveVariable('RMSD', rmsd_cv)
-        restraint_force.addGlobalParameter('lambda_restraints', 1.0)
+        restraint_force.addGlobalParameter('{}'.format(self.full_restraint_name), 1.0)
```

Comment:
Isn't `'{}'.format(self.full_restraint_name)` equivalent to just `self.full_restraint_name`?